app/serializers/current_user_serializer.rb

```diff
@@ -157,6 +158,10 @@ def muted_category_ids
     CategoryUser.lookup(object, :muted).pluck(:category_id)
   end
 
+  def ignored_users
+    IgnoredUser.where(user: object.id).joins(:ignored_user).pluck(:username)
```

Comment:
I understand that I might need to return the `id` too, but I haven't yet dove into the dimensions of how `incoming posts` work, so I want to keep it simple here.